src/Misc/fileTools.ts

```diff
@@ -13,6 +13,8 @@ import { ThinEngine } from '../Engines/thinEngine';
 import { EngineStore } from '../Engines/engineStore';
 import { Logger } from './logger';
 
+const base64DataUrlRegEx = new RegExp(/^data:([^,]+\/[^,]+)?;base64,/);

```

Comment:
I believe you still want the case invariant flag

```suggestion
const base64DataUrlRegEx = new RegExp(/^data:([^,]+\/[^,]+)?;base64,/i);
```

Reply:
Oops, forgot to push. I already fixed it :)